bindings/python/Makefile

```diff
@@ -16,9 +16,9 @@ install:
 	rm -rf prebuilt/win64/keystone.dll
 	rm -rf prebuilt/win32/keystone.dll
 	if test -n "${DESTDIR}"; then \
-		python setup.py build -b $(OBJDIR) install --root="${DESTDIR}"; \
+		python2 setup.py build -b $(OBJDIR) install --root="${DESTDIR}"; \
```

Comment:
any specific reason why python2 ?

Reply:
because install and install3 exist.
It would probably be more intuitive to have install and install2 with python3 as default, but changing the targets could potentially break some automatic setup. Not a decision I wanted to make.

Reply:
Ok, now this makes more sense. we need to rebuild the pip. the package manager is keep on dying

Reply:
what happens on older systems, where only `python` is available, but not `python2`?

Reply:
we might have more issue with if python  == python3 and not missing python2 ?

Reply:
Are there recent systems without python2 / 3? Seems like bad design to me.